src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/SearchDAO.scala

```diff
@@ -1,8 +1,21 @@
 package org.broadinstitute.dsde.firecloud.dataaccess
 
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import spray.json.JsObject
+
 /**
   * Created by davidan on 9/28/16.
   */
-trait SearchDAO {
+trait SearchDAO extends LazyLogging {
+
+  def initIndex
```

Comment:
I find it confusing that there is no function signature... is this the same as ```def initIndex(): Unit```?